docs/mp/testing.adoc

```diff
@@ -36,25 +47,77 @@ include::{rootdir}/includes/dependencies.adoc[]
 </dependency>
 ----
 
-== Usage - default
+== Usage
 A test can be annotated with `io.helidon.microprofile.tests.junit5.HelidonTest` annotation to mark it as a
```

Comment:
A test can be annotated with `io.helidon.microprofile.tests.junit5.HelidonTest` to mark it as

Reply:
fixed)